ekingan/d03-drills/css/reset.css

```diff
@@ -0,0 +1,47 @@
+/**
```

Comment:
@ekingan you don't actually have to check in the morning drills unless we ask you to.  Thanks though!  :camel: